XLS-0033d-CFTs/README.md

```diff
@@ -0,0 +1,836 @@
+
+###### 1.2.1.1.2.9. `OwnerNode`
+
+Identifies the page in the owner's directory where this item is referenced.
+
+##### 1.2.1.1.2. Example **`CFTokenIssuance`** JSON
+
+ ```json
+ {
+     "LedgerEntryType": "CFTokenIssuance",
+     "Flags" : 131072,
+     "Issuer" : "rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW",
+     "AssetCode": "5553440000000000000000000000000000000000",
+     "AssetScale": "2",
+     "MaximumAmount", "100000000",
+     "OutstandingAmount", "5",
+     "TransferFee": 50000,     
+     "CFTMetadata" : "",
+     "OwnerNode": "74"
+ }
+ ```
+ 
+##### 1.2.1.1.3. How do **`CFTokenIssuance`** objects work?
+
+Any account may issue up to 32 Compact Fungible Tokens, but each issuance must have a different **`AssetCode`**.
+
+###### 1.2.1.1.3.1. Searching for a **`CFTokenIssuance`** object
+
+CFT Issuances are uniquely identified by a combination of a type-specific prefix, the isser address and an asset code. To locate a specific **`CFTokenIssuance`**, the first step is to locate the owner directory for the issuer. Then, find the directory that holds `CFTokenIssuance` ledger objects and iterate through each entry to find the instance with the desired **`AssetCode`**. If that entry does not exist then the **`CFTokenIssuance`** does not exist for the given account.
+
+###### 1.2.1.1.3.2. Adding a **`CFTokenIssuance`** object
+
+A **`CFTokenIssuance`** object can be added by using the same approach to find the **`CFTokenIssuance`**, and adding it to that directory. If, after addition, the number of CFTs in the directory would exceed 32, then the operation must fail.
+
+###### 1.2.1.1.3.3. Removing a **`CFTokenIssuance`** object
+
+A **`CFTokenIssuance`** can be removed using the same approach, but only if the **`CurMintedAmount`** is equal to 0.
+
+###### 1.2.1.1.3.4. Reserve for **`CFTokenIssuance`** object
+
+Each **`CFTokenIssuance`** costs an incremental reserve to the owner account. This specification allows up to 32 **`CFTokenIssuance`** entries per account.
+
+#### 1.2.1.2. The **`CFToken`** object
+The **`CFToken`** object represents an amount of a token held by an account that is **not** the token issuer. CFTs are acquired via ordinary Payment or DEX transactions, and can optionally be redeemed or exchanged using these same types of transactions.
+
+##### 1.2.1.2.1 Fields
+A **`CFToken`** object can have the following required and optional fields. Notice that, unlike other objects, no field is needed to identify the object type or current owner of the object, because CFT holdings are grouped into pages that implicitly define the object type and identify the holder.
+
+| Field Name            | Required?          | JSON Type | Internal Type |
+| --------------------- |--------------------|-----------|---------------|
+| `CFTIssuanceID`       | :heavy_check_mark: |  `string` | `UINT256`     |
```

Comment:
should we call it `CFTokenIssuanceID` to make it consistent with rest of the naming?

Reply:
I think that's a good idea. @ledhed2222 can you make that change?